README.md

```diff
@@ -4,12 +4,23 @@ This research repository maintains the code and the results for the research pap
 
 _"Set theory has become the standard foundation for mathematics, as every mathematical object can be viewed as a set."_ -[Stanford Encyclopedia of Philosophy](https://plato.stanford.edu/entries/set-theory/)
 
-# Install Package from PyPI and Use
+# TL;DR Install Package from PyPI and Use
```

Comment:
No, `tl;dr` is used informally at the beginning of a very long text. It's not really appropriate here.

Reply:
These are not titles: `# Install package from PyPI`.

Move the code example to another section.